src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300px;
+}
+
+.search_bar {
+  margin-top: 20px;
+  position: relative;
+}
+
+.img {
+  position: absolute;
+}
+
+.enter {
+  outline: none;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.big_img {
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.big_search_bars {
+  height: 70px;
+  padding-left: 62px;
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.small_img {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.small_search_bars {
+  height: 42px;
+  padding-left: 33px;
+  font-family: inherit;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```

Comment:
you should have default search-bar and big/small. Not big and small at the same time. One of them is just search-bar without big/small word.